packages/webapp/package.json

```diff
@@ -51,7 +51,9 @@
         "@fluentui/react-focus": "8.8.8",
         "@fluentui/style-utilities": "8.8.1",
         "@fluentui/utilities": "8.13.2",
-        "@sap-ux/ui-components": "1.0.16",
-        "react-infinite-scroll-component": "6.1.0"
+        "@sap-ux/ui-components": "1.1.1",
+        "@types/react-transition-group": "4.4.5",
```

Comment:
Why do we need `react-transition-group` and if we need it, can we have the types as `devDependency`?

Reply:
Thanks, it's not required. I'll do a complete cleanup before submitting